src/lib/hint.ts

```diff
@@ -10,6 +10,9 @@ const texts = {
   enfants: (): string => {
     return `« Un enfant à charge » est un enfant dont vous êtes responsable et dont vous vous occupez, qu'il soit votre enfant naturel ou non.`
   },
+  types: (): string => {
```

Comment:
On est obligé de mettre `types` du fait du fonctionnement du `Hint`, mais c'est vraiment pas très claire je trouve si on arrive sur ce fichier 
> Ça a l'air d'être des steps, enfin sauf type qui est très générique

Je me dit qu'on pourrait rajouter un mini-comment pour expliquer 

```suggestion
  types: (): string => { // Step: resources/types
```

T'en pense quoi ?

Reply:
J'suis d'accord, j'ai pas trouvé ça hyper clair non plus en faisant la modif